File: rea2/app/src/main/java/com/jang/assistales/Fragment/HomeFragment.java
package com.jang.assistales.Fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

import com.jang.assistales.Common;
import com.jang.assistales.R;
import com.jang.assistales.model.CharacterBean;
import com.jang.assistales.model.DeitiesBean;
import com.jang.assistales.model.FaunaBean;
import com.jang.assistales.model.FloraBean;
import com.jang.assistales.model.ItemBean;
import com.jang.assistales.model.PlaceBean;
import com.jang.assistales.model.ProjectBean;
import com.jang.assistales.model.UniversBean;


public class HomeFragment extends Fragment {


    public HomeFragment() {
        // Required empty public constructor
    }


    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        FloraBean sheetBean1 = new FloraBean("Tulipe");
        FloraBean sheetBean2 = new FloraBean("Rose");
        FloraBean sheetBean3 = new FloraBean("MArguerite");
        ItemBean sheetBean4 = new ItemBean("Table");
        UniversBean sheetBean5 = new UniversBean("Andromède");
        CharacterBean sheetBean6 = new CharacterBean("Doe", "John");
        DeitiesBean sheetBean7 = new DeitiesBean("Indiens");
        FaunaBean sheetBean8 = new FaunaBean("chien");
        PlaceBean sheetBean9 = new PlaceBean("Paris");
        ProjectBean sheetBean10 = new ProjectBean("Livre1");

        Common.floraBeans.add(sheetBean1);
        Common.floraBeans.add(sheetBean2);
        Common.floraBeans.add(sheetBean3);
        Common.itemBeans.add(sheetBean4);
        Common.universBeans.add(sheetBean5);
        Common.characterBeans.add(sheetBean6);
        Common.deitiesBeans.add(sheetBean7);
        Common.faunaBeans.add(sheetBean8);
        Common.placeBeans.add(sheetBean9);
        Common.projectBeans.add(sheetBean10);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_home, container, false);
    }
}

File: rea5/assistales/app/src/main/java/com/jang/assistales/fragment/HomeFragment.kt
package com.jang.assistales.fragment

import android.content.Intent
import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.cardview.widget.CardView
import com.google.android.material.floatingactionbutton.FloatingActionButton
import com.jang.assistales.CreationActivity
import com.jang.assistales.MainActivity
import com.jang.assistales.R

class HomeFragment : Fragment() {

    private lateinit var cv_project: CardView
    private lateinit var cv_univers: CardView
    private lateinit var cv_character: CardView
    private lateinit var cv_deities: CardView
    private lateinit var cv_place: CardView
    private lateinit var cv_fauna: CardView
    private lateinit var cv_flora: CardView
    private lateinit var cv_item: CardView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        var view: View = inflater.inflate(R.layout.fragment_home, container, false)
        cv_project = view.findViewById(R.id.cv_project)
        cv_project.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 1)
            startActivity(intent)
        }
        cv_univers = view.findViewById(R.id.cv_univers)
        cv_univers.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 2)
            startActivity(intent)
        }
        cv_character = view.findViewById(R.id.cv_character)
        cv_character.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 3)
            startActivity(intent)
        }
        cv_deities = view.findViewById(R.id.cv_deities)
        cv_deities.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 4)
            startActivity(intent)
        }
        cv_place = view.findViewById(R.id.cv_place)
        cv_place.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 5)
            startActivity(intent)
        }
        cv_fauna = view.findViewById(R.id.cv_fauna)
        cv_fauna.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 6)
            startActivity(intent)
        }
        cv_flora = view.findViewById(R.id.cv_flora)
        cv_flora.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 7)
            startActivity(intent)
        }
        cv_item = view.findViewById(R.id.cv_item)
        cv_item.setOnClickListener { view ->
            val intent = Intent(activity, MainActivity::class.java)
            intent.putExtra("type", 8)
            startActivity(intent)
        }
        return view
    }


}